crates/miden-lib/asm/miden/note.masm

```diff
@@ -336,3 +339,36 @@ export.add_note_assets_to_account
     # clear the stack
     drop dropw drop
 end
+
+#! Writes the assets data stored in the advice map to the memory by the provided destination
+#! pointer. 
+#!
+#! Inputs:
+#!   Operand stack: [ASSETS_COMMITMENT, num_assets, note_index, dest_ptr]
+#!   Advice map: {
+#!     ASSETS_COMMITMENT: [[ASSETS_DATA]]
+#!   }
+#! Outputs:
+#!   Operand stack: [num_assets, note_index, dest_ptr]
+#!   Advice stack: []
```

Comment:
nit: it's a bit odd that we are showing the state of the advice stack here but we don't do this for inputs. Since this procedure doesn't change the state of the advice stack, I don't think we actually need to show it.

Reply:
Agreed, I just forget to remove it.